lua/terrortown/lang/en.lua

```diff
@@ -1959,7 +1959,7 @@ L.help_hud_elements_special_settings = "These are specific settings for the used
 
 -- 2023-10-25
 L.help_keyhelp = [[
-Key bind helpers are part of a UI element that always shows relevant keybindings to the player, which is especially helpful for new players. There are three different types of key bindings:
+Key bind helper is an UI element that always shows relevant keybindings to the player, which is especially helpful for new players. There are three different types of key bindings:
```

Comment:
```suggestion
Key bind helper is a UI element that always shows relevant keybindings to the player, which is especially helpful for new players. There are three different types of key bindings:
```

it is "a UI" not "an UI"

Reply:
I apply this change myself, so I can merge it

Reply:
Ah... right... I forgot that in some cases there are some exceptions when using "an" before U or O.
Thanks for noticing it. 👍